src/app/vulnerabilities/404with200statuscode.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
You can merge these files under one, and add comments to define which function does what